Sources/SwiftSyntaxBuilder/gyb_generated/Buildables.swift

```diff
@@ -0,0 +1,8325 @@
+    let memberAccessExpr = buildMemberAccessExpr(format: format, leadingTrivia: leadingTrivia)
+    return ExprSyntax(memberAccessExpr)
+  }
+}
+
+public struct IsExpr: ExprBuildable {
+  let isTok: TokenSyntax
+  let typeName: TypeBuildable
+
+  public init(
+    isTok: TokenSyntax,
+    typeName: TypeBuildable
+  ) {
+    self.isTok = isTok
+    self.typeName = typeName
+  }
+  
+  func buildIsExpr(format: Format, leadingTrivia: Trivia?) -> IsExprSyntax {
+    let isExpr = SyntaxFactory.makeIsExpr(
+      isTok: isTok,
+      typeName: typeName.buildType(format: format, leadingTrivia: nil)
+    )
+    
+    if let leadingTrivia = leadingTrivia {
+      return isExpr
+        .withLeadingTrivia(leadingTrivia)
+    }
+
+    return isExpr
+  }
+
+  public func buildExpr(format: Format, leadingTrivia: Trivia?) -> ExprSyntax {
+    let isExpr = buildIsExpr(format: format, leadingTrivia: leadingTrivia)
+    return ExprSyntax(isExpr)
+  }
+}
+
+public struct AsExpr: ExprBuildable {
+  let asTok: TokenSyntax
+  let questionOrExclamationMark: TokenSyntax?
+  let typeName: TypeBuildable
+
+  public init(
+    asTok: TokenSyntax,
+    questionOrExclamationMark: TokenSyntax?,
+    typeName: TypeBuildable
+  ) {
+    self.asTok = asTok
+    self.questionOrExclamationMark = questionOrExclamationMark
+    self.typeName = typeName
+  }
+  
+  func buildAsExpr(format: Format, leadingTrivia: Trivia?) -> AsExprSyntax {
+    let asExpr = SyntaxFactory.makeAsExpr(
+      asTok: asTok,
+      questionOrExclamationMark: questionOrExclamationMark,
+      typeName: typeName.buildType(format: format, leadingTrivia: nil)
+    )
+    
+    if let leadingTrivia = leadingTrivia {
+      return asExpr
+        .withLeadingTrivia(leadingTrivia)
+    }
+
+    return asExpr
+  }
+
+  public func buildExpr(format: Format, leadingTrivia: Trivia?) -> ExprSyntax {
+    let asExpr = buildAsExpr(format: format, leadingTrivia: leadingTrivia)
+    return ExprSyntax(asExpr)
+  }
+}
+
+public struct TypeExpr: ExprBuildable {
+  let type: TypeBuildable
+
+  public init(
+    type: TypeBuildable
+  ) {
+    self.type = type
+  }
+  
+  func buildTypeExpr(format: Format, leadingTrivia: Trivia?) -> TypeExprSyntax {
+    let typeExpr = SyntaxFactory.makeTypeExpr(
+      type: type.buildType(format: format, leadingTrivia: nil)
+    )
+    
+    if let leadingTrivia = leadingTrivia {
+      return typeExpr
+        .withLeadingTrivia(leadingTrivia)
+    }
+
+    return typeExpr
+  }
+
+  public func buildExpr(format: Format, leadingTrivia: Trivia?) -> ExprSyntax {
+    let typeExpr = buildTypeExpr(format: format, leadingTrivia: leadingTrivia)
+    return ExprSyntax(typeExpr)
+  }
+}
+
+public struct ClosureCaptureItem: SyntaxBuildable {
+  let specifier: TokenListSyntax?
+  let name: TokenSyntax?
+  let assignToken: TokenSyntax?
+  let expression: ExprBuildable
+  let trailingComma: TokenSyntax?
+
+  public init(
+    specifier: TokenListSyntax?,
```

Comment:
Should we add `specifier: TokenListSyntax? = nil` here so it is possible to skip the parameters?